scripts/newinstall.sh

```diff
@@ -118,23 +135,21 @@ if true; then
 			The git version control system is frequently used with LSST software. While
 			the LSST stack should build and work even in the absence of git, we don't
 			regularly run and test it in such environments. We therefore recommend you
-			have at least git 1.8.4 installed, or let this script install it for you.
-
-			git installed by this installer will be managed by LSST's EUPS package manager,
-			and will not replace or modify any other version of git you may have installed
-			on your system.
+			have at least git 1.8.4 installed with you normal package manager.
 
 			EOF
 
 			while true; do
-				read -p "Would you like us to install git for you (if unsure, say yes)? " yn
+				read -p "Would you like to try continuiung without git? " yn
```

Comment:
I think we need to propose a default for a user who doesn't know what is the right answer. A number of our users are non-experts who may get stumped by this question. I'm fine with the default being 'no', FYI.